hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/spi/impl/ReconDBProvider.java

```diff
@@ -138,6 +138,14 @@ private static DBStore initializeDBStore(OzoneConfiguration configuration,
 
   public void close() throws Exception {
     if (this.dbStore != null) {
+      try {
+        // Flush to ensure all data is persisted to disk before closing.
+        dbStore.flushDB();
```

Comment:
we can move this inside dbStore.close() as its part of RDBStore, applicable to all flow.